htdocs/accountancy/journal/bankjournal.php

```diff
@@ -433,6 +433,9 @@
 					$paymentsalstatic->label = $links[$key]['label'];
 					$tabpay[$obj->rowid]["lib"] .= ' '.$paymentsalstatic->getNomUrl(2);
 					$tabpay[$obj->rowid]["paymentsalid"] = $paymentsalstatic->id;
+					if ($compta_user) {
+						$tabtp[$obj->rowid][$compta_user] += $amounttouse;
```

Comment:
The $tabtp should be already filled previously by line 359 in the

elseif ($links[$key]['type'] == 'user')

but may be we don't enter into it because of line 318 ?
of because into database we don't have the line for the user ?

Reply:
Thanks @eldy for your return.

> _The $tabtp should be already filled previously by line 359_:

I'm ok with this but no longer applicable for unknown reasons

> _may be we don't enter into it because of line 318 ?_:

This line is here to exclude the user from social contribution, if present, otherwise adding a line we don't need

> _We don't have the line_:

No, the user line is there in llx_bank_url

![image](https://github.com/Dolibarr/dolibarr/assets/2341395/b2cddf24-d27c-414e-af81-1ef80eabed22)

Reply:
So if line "user" is here, it means that after the PR, the amount will be added twice:
- once on line 373:   $tabtp[$obj->rowid][$compta_user] += $amounttouse;
- once on the new added line 437:   $tabtp[$obj->rowid][$compta_user] += $amounttouse;

same code, but executed twice because we loop on $links that is list of all linked lines. Can you check if $links is correctly loaded with the line 77 (and not just the line 76) ?

Reply:
@aspangaro A feedback on my last comment ?